spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/route/SimpleHostRoutingFilter.java

```diff
@@ -245,6 +245,7 @@ protected CloseableHttpClient newClient() {
 			httpClientBuilder.setSSLHostnameVerifier(NoopHostnameVerifier.INSTANCE);
 		}
 		return httpClientBuilder.setConnectionManager(newConnectionManager())
+				.disableContentCompression()
```

Comment:
This is the original fix #1372. Again, why does this PR exist? We can just cherry-pick.